cmake/project_common.cmake

```diff
@@ -31,15 +31,19 @@ if(WIN32 AND NOT WATCOM)
     endif()
 
     set(EXTRA_LIBS gdi32.lib winspool.lib shell32.lib ole32.lib comdlg32.lib advapi32.lib)
+    set(WINCON_WINGUI_DEP_LIBS winmm.lib)
     set(SDL2_DEP_LIBRARIES version.lib winmm.lib imm32.lib)
 elseif(WATCOM_WIN32)
     set(EXTRA_LIBS "winmm.lib")
+    set(WINCON_WINGUI_DEP_LIBS "")
```

Comment:
Shouldn't the winm.lib be moved from "extra" to wincon_wingui here?

Reply:
I'm not sure about the Watcom compiler. I've never used it. Since winmm.lib was in the extra libs for Watcom before, I figured that it must be needed for all of ports so I didn't want to move it to the WINCON/WINGUI libs list.

Of course if you know that winmm.lib is needed only for WINCON/WINGUI when building with Watcom, then we should move it. I just don't know and didn't want to change the existing setup.

Reply:
Just give it a try and we see if travis/appveyor look better or worse :-)

Reply:
The `winmm.lib` dependency is somewhat recent,  from the commit 78e6949 mentioned above.  It applies to only WinCon/WinGUI.  Both of those now call `PlaySound`.  VT (and other platforms) don't.

Thanks for tackling this.  I know nearly nothing about CMake,  so I'm quite happy when somebody else pitches in on it.

Reply:
Ok, `winmm.lib` is moved to the WINCON/WINGUI libs for the Watcom compiler. Hopefully we get a clean bill of health from Appveyor/Travis, but from what Bill said, it looks like that will be the case.